_test/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(TEST_DIRECTORIES
+    "test_data_loaders"
+    "test_config"
+    "test_IX_classes"
+    "test_map_mask"
+    "test_mslice_objects"
+    "test_multifit"
+    "test_multifit_legacy"
+    "test_utilities"
+    "test_instrument_classes"
+    "test_docify"
+    "test_admin"
+    "test_mpi"
+)
+
+foreach(_test_dir ${TEST_DIRECTORIES})
+    matlab_add_unit_test(
+        NAME "Matlab.${_test_dir}"
+        CUSTOM_TEST_COMMAND
```

Comment:
defining this inline is inelegant; a function with `_test_dir` as an argument would be neater